about.html

```diff
@@ -0,0 +1,168 @@
+      <ul class="nav-menu ml-auto">
+        <li class="nav-items">
+          <a href="#about" class="nav-links about">About</a>
+        </li>
+        <li class="nav-items">
+          <a href="#teachers" class="nav-links">Teachers</a>
+        </li>
+        <li class="nav-items">
+          <a href="#partners" class="nav-links">Partners</a>
+        </li>
+        <li class="nav-items">
+          <a href="#contact" class="nav-links">Contact</a>
+        </li>
+        <li class="nav-items">
+          <a href="#getclass" class="nav-links get">Get a Class</a>
+        </li>
+      </ul>
+      <div class="hamburger">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </div>
+  </header>
+  <section class="head ">
+    <div class="container">
+    <div class="jumbotron">
+      <h1 class="jumbotron-center text-center">First Crypto Learning <br>
+        Complete Course</h1>
+      <p class="about-note text-center">
+        The Global Summit brings together the community of experts, academics, and activists who comprise the Creativity Commons affiliate network in a different country every two years. <br>
+        Welcome to our website. After 7 monthes you will be one of the most professional crypto trader and analysis in
+        the world.
+         free software advocates Wikipedians. Open Knowledge, gallereis, libraies, musemus, archies, and many more
+      </p>
+      <p class="please text-center">Contact Email about CC Global Summit 2015!</p>
+      <p class="email text-center">crypto@pasqad.com ></p>
+    </div>
+  </div>
+  </section>
+  <section class="intro">
+    <div class="container">
+      <h2 class="text-center intro-titel">
+        Creative Common Crypto Center
+      </h2>
+      <hr class="line">
+      <p class="about-note text-center">
+        The SEC's decision to expand its cryptocurrency unit has been praised by industry experts, with Dr. Anna Becker,
+        CEO and co-founder of EndoTech, calling it "a welcome development." She believes that enhanced security,
+        regulation and complex financial investment solutions will enable digital currencies to become more accepted.
+  
+      </p>
+  
+      <div class=" text-center">
+        <img src="assets/img/partners/Infobaelogo.png" alt="info logo"
+          class="infobael">
+      </div>
+  
+    </div>
+  </section>
+  <section class="intro">
+    <div class="container">
+      <h2 class="text-center intro-titel">
+        Crypto Learnings
+      </h2>
+      <hr class="line">
+      <ul class="pevents">
+        <li class="pevent one text-center">
+            <h3>2010</h3>
+            <p>Microverse 2018 in California</p>
+        </li>
+        <li class="pevent two text-center">
+            <h3>2013</h3>
+            <p>Microverse 2019 in California</p>
+        </li>
+    </ul>
```

Comment:
![image](https://user-images.githubusercontent.com/60612329/170841699-1ecb4813-2575-436f-99cc-ea661f6669df.png)
currently, your text on this page are crashing into each other please implement some padding or margin to improve the UX of the webpage

Reply:
Thank you for the time to review my code. I checked this section in mobile version but it completely OK in any browser that I checked.
![image](https://user-images.githubusercontent.com/33055346/170841848-42fecac0-5028-4c05-8866-a0933da5ff3e.png)